Completed test case C15

developersbm opened the pull request:
Implemented test C15 in readingsCompareMeterFlow.js

Co-authored by:
Sebastian Bastida Marin: @developersbm
Aliyah McDowell: @amcdowell226
Colin Nguyen: @khoamelo

Partly Addresses #962

Type of change
Implemented test for C15
(Check the ones that apply by placing an "x" instead of the space in the [ ] so it becomes [x])

[ ] Note merging this changes the database configuration.
[ ] This change requires a documentation update

Checklist
(Note what you have done by placing an "x" instead of the space in the [ ] so it becomes [x]. It is hoped you do all of them.)

[x] I have followed the OED pull request ideas
[x] I have removed text in ( ) from the issue request
[x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

Limitations
The implemented test doesn't work with C15's expected values [1990.55774277443, 2057.611897078],  but works with the expected values [7962.23097109771, 8230.447588312], which is the expected value for C2.

developersbm committed:
Completed test case C15

developersbm committed:
Completed test case C15

huss committed:
Merge remote-tracking branch 'origin/development' into pr/developersbm/1375

huss committed:
update expexted values

With the merging of PR #1362, the expected values in the testing
design document can now be used.

huss committed:
have file consistently use tabs

huss commented:
I merged in the updated SQL (merged development) and updated the expected values to the new ones in the testing design doc. They all work now. I also needed to update the expected value for CG15 due to the value being wrong in the testing design document. CG15 is not really part of this PR put the tests were failing so I included this fix in this PR. See the developer Discord channel for a discussion of the issue.

huss committed:
fix CG15 expected values

This is not really part of this PR but needed to get this test to pass.
The testing design document had the wrong values for compare
group flow so when the new SQL was merged it started to fail.

huss reviewed (approved):
Thanks to @developersbm, @amcdowell226 & @khoamelo for their first contribution to OED. Review and testing found it was implemented as desired. See my comments/commits for some changes I made due to issues with the testing design doc. Sorry about them. Congratulations on your first accepted work into OED.
